perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2523,25 +2525,38 @@ private String createUserAccountInExternalSystem (PerunSession session, User use
 	}
 
 	/**
-	 * Execute process which expires member and removes his sponsorship
+	 * Execute process which expires member and removes his sponsorship.
 	 *
 	 * @param sess perun session
 	 * @param sponsoredMember who will be processed
+	 * @param vo of sponsored member
 	 */
-	private void processMemberAfterRemovingLastSponsor(PerunSession sess, Member sponsoredMember) {
+	private void processMemberAfterRemovingLastSponsor(PerunSession sess, Member sponsoredMember, Vo vo) {
 
-		//Set member's expiration to today and set status to expired.
+		//Check whether the member should be expired according to vo's expiration rules.
+		boolean cannotBeExpired = false;
```

Comment:
I would change this variable to "shouldBeExpired = true". It will be easier for reading.

Reply:
Done